src/udev/net/link-config.c

```diff
@@ -701,6 +702,40 @@ int config_parse_ifalias(
         return 0;
 }
 
+int config_parse_txqueuelen(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        uint32_t k, *v = data;
+        int r;
+
+        *v = (uint32_t) -1;
```

Comment:
Please drop this. If an invalid string is assigned, then we should not change anything.

Reply:
Will be dropped.